### ISSUE TITLE
Order content type and management screens

### ISSUE THREAD

**Issue by bleroy** [OrchardCMS/OrchardCore.Commerce#18]
Order management should rely on workflows, in order to allow rich configurability.

Orders are content items. The order part has the following properties:

* string or enumeration Status (should this just be an enum field?)
* Customer
* Currency
* Line items (see [shopping cart](https://github.com/OrchardCMS/OrchardCore.Commerce/issues/15) for definition)

Nwazet had a tracking URL on the order part, but that should be handled somewhere else, likely shipping: some orders may be split into several packages, each with its tracking URL.

Nwazet had an activity text field. This doesn't need to be built into the part, and can instead be a standard text field.

**Comment by MatteoPiovanelli-Laser** [comment]
> Nwazet had a tracking URL on the order part, but that should be handled somewhere else, likely shipping: some orders may be split into several packages, each with its tracking URL.

I agree. If there were an interface to inject additional information into the order this would be an ideal implementation.

Another thing that came up, and I think workflows can take care of, is the fact that some orders may not immediately be valid, but may require some action from a back-office operator.

**Comment by DrewBrasher** [comment]
I will work on this.

**Comment by DrewBrasher** [comment]
What would be the best way for me to add an order to test with?  I was going to do it by going through the checkout process but got an error adding an item to the cart so I'm not sure if I just don't have everything setup right or if that process is not completed yet.  Would it be better if I created a recipe that creates some products and an order?

**Comment by bleroy** [comment]
Please file a bug with a repro for the shopping cart problem, but that won't help you create an order anyway, since checkout is not done. Instead, just create an order from the admin, it's just a content type. I think there's a recipe for creating the order type. LMK if that's not the case, can't check right now, but I can check later. Shouldn't be too hard to create if that's not the case.

**Comment by DrewBrasher** [comment]
I created issue #73  for the shopping cart problem.  You are correct that there is a recipe for creating the order content type.  I am able to create an order from the admin but I don't see how I can add Items to the order.

**Comment by bleroy** [comment]
You can't add items to the order as it is.
